indigo/distribution.yaml

```diff
@@ -10075,6 +10075,22 @@ repositories:
       url: https://github.com/GT-RAIL/rail_collada_models.git
       version: develop
     status: maintained
+  rail_face_detector:
+    doc:
+      type: git
+      url: https://github.com/GT-RAIL/rail_face_detector.git
+      version: master
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/gt-rail-release/rail_face_detector-release.git
+      version: 0.0.2-0
+    source:
+      test_pull_requests: true
```

Comment:
just checking: Did you follow the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.

Reply:
I think that may have been a mistake in the build release. Which line should I remove?

Reply:
line 10089 `test_pull_requests: true`